nixos/modules/system/boot/stage-1-init.sh

```diff
@@ -301,6 +301,7 @@ mountFS() {
         *x-nixos.autoresize*)
             if [ "$fsType" = ext2 -o "$fsType" = ext3 -o "$fsType" = ext4 ]; then
```

Comment:
This line is not part of this PR, so suggesting improvements to it is kind of off-topic for this PR.